src/components/shared/custom-button.tsx

```diff
@@ -0,0 +1,22 @@
+import Button from '@mui/material/Button';
+import { styled } from '@mui/material/styles';
+
+interface CustomTypes {
```

Comment:
all props interfaces should name like components. fix everywhere. CustomButtonProps